appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 5.2.7.{build}
+version: '{build}'
```

Comment:
In attempt to minimize the frictions. I see no point in this - appveyour shows branch or tag if available. IMHO that's quite enough:
![image](https://user-images.githubusercontent.com/22025304/80360545-2cfe1400-8888-11ea-9517-eb4bd68ce000.png)